pext/VERSION

```diff
@@ -1 +1 @@
-0.32
\ No newline at end of file
+0.32-146-g87658a1
```

Comment:
Please don't commit this file

Reply:
it would be nice if something prevented this

personally i like to bring people/projects over to setuptools_scm but that may not be a good fit for pext yet